substrate/frame/staking/src/tests.rs

```diff
@@ -5416,8 +5416,10 @@ fn count_check_works() {
 	})
 }
 
+// TODO: should panic once <https://github.com/paritytech/polkadot-sdk/issues/3245> is fixed.
 #[test]
-#[should_panic = "called `Result::unwrap()` on an `Err` value: Other(\"number of entries in payee storage items does not match the number of bonded ledgers\")"]
+//#[should_panic = "called `Result::unwrap()` on an `Err` value: Other(\"number of entries in payee
+//#[should_panic storage items does not match the number of bonded ledgers\")"]
 fn check_payee_invariant1_works() {
```

Comment:
I think you can also add something like `#[cfg(feature = "unused")]` but its not a big difference.